sentry-core/src/main/java/io/sentry/core/SentryClient.java

```diff
@@ -79,7 +79,17 @@ public SentryClient(final @NotNull SentryOptions options, @Nullable Connection c
     }
 
     for (EventProcessor processor : options.getEventProcessors()) {
```

Comment:
I realize this code was here already, so maybe a separate PR but we should just run the processors once (have have the same code on `applyScope` below using processors from the scope. We could merge the lists (run scope processors last). I'll raise an issue.